docs/running-on-yarn.md

```diff
@@ -134,6 +134,13 @@ Most of the configs are the same for Spark on YARN as for other deployment modes
   </td>
 </tr>
 <tr>
+  <td><code>spark.yarn.am.port</code></td>
+  <td>(random)</td>
+  <td>
+    Port for the YARN Application Master to listen on.
```

Comment:
I think we should be more specific in the description of this.  an application master could have many ports open, what is this one for.  I think either put in name or atleast put in description.